markets/perps-market/test/integration/helpers/collateralHelper.ts

```diff
@@ -0,0 +1,121 @@
+import { ethers } from 'ethers';
+import { Systems } from '../bootstrap';
+
+type IncomingChainState = {
+  systems: () => Systems;
+  provider: () => ethers.providers.JsonRpcProvider;
+};
+
+type TradeSynthData = {
+  synthName: () => string;
```

Comment:
dont think we need this. also we should just pass the entire synth that gets returned from `bootstrapSynthMarkets` so that way you have the id and the synth itself.  instead of destructuring